site/src/planck-io.md

```diff
@@ -134,20 +137,29 @@ Delete file `f`.
 
 Spec<br/>
  _args_: `(cat :f (or :string string? :file file?))`<br/>
- 
-### <a name=""></a>directory?
+
+### <a name="directory?"></a>directory?
```

Comment:
@mfikes I think this anchor was missing - please confirm.

Reply:
Yeah, that link is definitely broken in the online site documentation.